packages/react-native-renderer/src/ReactNativeSetNativeProps.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ */
+
+import {create} from './ReactNativeAttributePayload';
+import {warnForStyleProps} from './NativeMethodsMixinUtils';
+
+import warningWithoutStack from 'shared/warningWithoutStack';
+
+// Module provided by RN:
+import UIManager from 'UIManager';
+
+export function setNativeProps(handle: any, nativeProps: Object) {
```

Comment:
Is there a better flow type I should be using for `handle`?

Reply:
We used to pretend that native host instances were ReactNativeComponent. That’s not technically true because they’re not `instanceof`.

With forwardRef, we get the inner type. I would’ve expected that you have to supply the ref type to Flow. What does Flow think the type of a ref is?

Ideally it would be an opaque type which will help catch anyone trying to call something else on it.

Reply:
Alright, seems like a problem for another PR at another time. :)